docs/commands.md

```diff
@@ -97,11 +97,12 @@ Arguments:
 Result: Warns `warnUser` with a DM and adds `points` points to their points total.
```

Comment:
I can't comment up there but the arguments section & this description are both out of date.